pset-iterators.js

```diff
@@ -114,7 +121,11 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+  return president.took_office === '1850-1900';
```

Comment:
@deedeemedina you're on the right path, the goal of this problem was to extract the year a president took office and then do an in-between comparison to confirm if the tear they took office year was in-between 1850 and 1900. 

Take a look at the solution for more details:  https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L124